mmv1/products/bigqueryreservation/api.yaml

```diff
@@ -70,3 +70,12 @@ objects:
           If false, any query using this reservation will use idle slots from other reservations within
           the same admin project. If true, a query using this reservation will execute with the slot
           capacity specified above at most.
+      - !ruby/object:Api::Type::Integer
+        name: 'concurrency'
+        description: |
+          Maximum number of queries that are allowed to run concurrently in this reservation. This is a soft limit due to asynchronous nature of the system and various optimizations for small queries. Default value is 0 which means that concurrency will be automatically set based on the reservation size.
+      - !ruby/object:Api::Type::Boolean
+        name: 'multiRegionAuxiliary'
```

Comment:
Are we able to add this to a test?

Reply:
id like to avoid this as this applies to EU/US and might cause issues with on region level for other tests